iree/compiler/Conversion/LinalgToSPIRV/Passes.cpp

```diff
@@ -171,25 +169,8 @@ void buildSPIRVTransformPassPipeline(OpPassManager &pm,
   //   - All Linalg/Loops/GPU/Affine/Standard ops are converted away.
   //   - The module contains the final spv.module ready for serialization.
   //===--------------------------------------------------------------------===//
-  addLinalgToSPIRVPasses(pm, options);
+  buildLinalgToSPIRVPassPipeline(pm, options);
 }
 
-static PassPipelineRegistration<> linalgToSPIRVPipeline(
```

Comment:
Still going through the change, but are these command line options defined somewhere?

Reply:
I see they are added later on.